pi-manage

```diff
@@ -29,1714 +29,27 @@
 #            Add sub command for policies
 # 2014-12-15 Cornelius Kölbel, info@privacyidea.org
 #            Initial creation
+
+# SPDX-FileCopyrightText: (C) 2023 Paul Lettich
```

Comment:
Is this correct from a copy right perspective?

Reply:
@cornelinux This is based on the [Reuse-Project](https://reuse.software/) for machine-readable license information.
Probably there should be a line for each author and year but i was lazy, that is why i kept changelog.